disnake/sticker.py

```diff
@@ -128,7 +128,11 @@ class _StickerTag(Hashable, AssetMixin):
     @property
     def url(self) -> str:
         """:class:`str`: The url for the sticker's image."""
-        return f"{Asset.BASE}/stickers/{self.id}.{self.format.file_extension}"
+        # https://github.com/discord/discord-api-docs/issues/6675#issuecomment-1954755672
+        base = (
+            "https://media.discordapp.net" if self.format is StickerFormatType.gif else Asset.BASE
```

Comment:
maybe put the url in a variable like Asset.BASE to stay consistent with the old

Reply:
Unlike `Asset.BASE´, this is only used in a single location so far, and will hopefully remain a rare edge case :/
Nonetheless, I don't see why not - moved in 8a00106de122b2500480bd9f31e6d59445153b0a.